js/script.js

```diff
@@ -1,205 +0,0 @@
-                                        </circle>
-                                    </svg>
-                                <figure>`;
-        ajaxStatus = "loading";
-      }
-      if (status == "success") {
-        messageText = `<div class="success-box">
-                                    <svg class="checkmark checkmark-success" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 52 52">
-                                        <circle class="checkmark-circle" cx="26" cy="26" r="25" fill="none"/>
-                                            <path class="checkmark-check" fill="none" d="M14.1 27.2l7.1 7.2 16.7-16.8"/>
-                                    </svg>
-                                    <strong>Sua mensagem foi enviada!</strong>
-                                    <div class="column is-narrow has-text-centered">
-                                        <button type="button" title="retornar" class="button is-info back-button">Retornar</button>
-                                    </div>
-                                </div>`;
-        ajaxStatus = "success";
-      }
-      if (status == "fail") {
-        messageText = `<div class="error-box">
-                                    <svg class="checkmark  checkmark-error" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 52 52">
-                                        <circle class="checkmark-circle" cx="26" cy="26" r="25" fill="none"/>
-                                        <path class="checkmark-check" fill="none" d="M16 16 36 36 M36 16 16 36"/>
-                                    </svg>
-                                    <strong>Ocorreu um erro :( <br/> Tente novamente mais tarde</strong>
-                                    <div class="column is-narrow has-text-centered">
-                                        <button type="button" title="retornar" class="button is-info back-button">Retornar</button>
-                                    </div>
-                                </div>`;
-        ajaxStatus = "fail";
-      }
-
-      msgBox = `<div class="message-alert">
-                        <div class="is-${ajaxStatus}">
-                                ${messageText}
-                        </div>
-                     </div>`;
-
-      return msgBox;
-    },
-
-    updateMsg: (nodes, msg) => {
-      nodes.insertAdjacentHTML("beforeend", msg);
-    },
-
-    return: () => {
-
-      const form = document.querySelector(".form");
-      const fields = document.querySelectorAll(".input-field");
-      const messageBox = document.querySelectorAll(".message-alert");
-
-      form.style.display = "block";
-
-      fields.forEach(field => {
-          field.value = "";
-      });
-
-      messageBox.forEach(box =>{
-        box.remove();
-      });
-    },
-
-    sendData: e => {
-      e.preventDefault();
-
-      const initFunctions = startForm.functions;
```

Comment:
if you gonna name this initFunctions then you should find a better name for the startForm property.
Maybe startForm.initFunctions, so you can destruct it here (and no one will get confused):
```
const {initFunctions} = startForm
```